app/Models/User.php

```diff
@@ -50,6 +52,13 @@ class User extends Authenticatable
         'email_verified_at' => 'datetime',
     ];
 
+    public static function getAllAsOptions(): Collection
+    {
+        return self::select('id', 'name')
+            ->where('organization_id', Auth::user()->organization_id)
+            ->get();
+    }
+
```

Comment:
this will be used for task subscriber and owner dropdowns. Only want users who are also part of the same org or else you would be getting random people from anywhere